src/Tizen.Wearable.CircularUI.Forms.Renderer/FlatViewCellRenderer.cs

```diff
@@ -0,0 +1,32 @@
+﻿/*
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Flora License, Version 1.1 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://floralicense.org/license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using Tizen.Wearable.CircularUI.Forms;
+using Tizen.Wearable.CircularUI.Forms.Renderer;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.Tizen;
+
+[assembly: ExportRenderer(typeof(FlatViewCell), typeof(FlatViewCellRenderer))]
+namespace Tizen.Wearable.CircularUI.Forms.Renderer
+{
+    public class FlatViewCellRenderer : ViewCellRenderer
+    {
+        public FlatViewCellRenderer()
+        {
+            Style = "full_effect_off";
+        }

```

Comment:
```suggestion
        public FlatViewCellRenderer() : base("full_effect_off")
        {
        }

Reply:
base class is `ViewCellRenderer` which doesn't take a style argument. You think it's better to create a new `ViewCellRenderer(string style) : base(style)` constructor?

Reply:
nono. it's ok. leave it.